apps/aehyperchains/src/aehc_consensus_hyperchains.erl

```diff
@@ -173,12 +174,11 @@ assert_config(_Config) ->
     case aeu_env:user_config([<<"hyperchains">>, <<"activation_criteria">>]) of
         undefined -> ok;
         {ok, Criteria} ->
-%%            lager:debug("Trying to set the activation criteria")
             MinStake = maps:get(<<"minimum_stake">>, Criteria),
             MinDelegates = maps:get(<<"unique_delegates">>, Criteria),
             BlockFreq = maps:get(<<"check_frequency">>, Criteria),
             BlockConfirms = maps:get(<<"confirmation_depth">>, Criteria),
-            ok = set_hc_activation_criteria(MinStake, MinDelegates, BlockFreq, BlockConfirms)
+            _ = set_hc_activation_criteria(MinStake, MinDelegates, BlockFreq, BlockConfirms)
```

Comment:
Looks like rebase error. This assert should be here.

Reply:
ok, fixed